ts/services/PolicyService.ts

```diff
@@ -14,12 +14,17 @@ class PolicyService {
         json: true,
         uri: config.endpoint + '/policies'
       });
-      logger.info('The following policies are installed:');
-      logger.info(response.policies.sort((pol1, pol2) => {
-        return pol1.name.localeCompare(pol2.name);
-      }).map((pol) => {
-        return pol.name + ' (ID: ' + pol.id + ')';
-      }));
+      if (config.format === 'text') {
+        logger.info('The following policies are installed:');
+        logger.info(response.policies.sort((pol1, pol2) => {
+          return pol1.name.localeCompare(pol2.name);
+        }).map((pol) => {
+          return pol.name + ' (ID: ' + pol.id + ')';
+        }));
+      }
+      if (config.format === 'json') {
+        logger.info(JSON.stringify(response, null, 2));
```

Comment:
When formatting to JSON do you want to still sort it in some way? I'm thinking to when we want to export packages into a local directory for git tracking. Ordering will matter to make sure there aren't unnecessary tracked changes from reordering of items.

Reply:
I think it's a good consideration.  I opened #37 an issue to track for later.  Right now this works and is consistent with others.